### PR TITLE
Test rootsquash volume set operation

### PR DESCRIPTION
**Pull request by msainiRedhat**
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by msainiRedhat** [comment]
Logs - http://magna002.ceph.redhat.com/ceph-qe-logs/msaini/Automation/CEPH-83581040/cepci-log/

File: tests/nfs/nfs_export_rootsquash_windows.py [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        )
+
+        # Redeploy nfs
+        CephAdm(installer).ceph.orch.redeploy(service=f"nfs.{nfs_name}")
+
+        # Add wait till the NFS daemons are up
+        sleep(10)
+
+        # Mount the volume with rootsquash enable on client
+        sleep(9)
```
Comment (pranavprakash20) [inline tests/nfs/nfs_export_rootsquash_windows.py:109]: Are these two sleeps needed @msainiRedhat 
If yes, can we club it?

**Review by pranavprakash20 — APPROVED** [review]
LGTM

**Review by MohitBis — APPROVED** [review]
LGTM

**Review by vdas-redhat — APPROVED** [review]
LGTM

**Pull request merged**